JavaScript/upscript.js

```diff
@@ -0,0 +1,95 @@
+document.addEventListener('DOMContentLoaded', function() {
+    const emailInput = document.getElementById('email');
+    const emailError = emailInput.nextElementSibling;
```

Comment:
P2:
해당 변수명만 봤을때는 요소의 타입을 알기가 어렵습니다. 에러 메시지처럼 생각되는 변수명이니 
조금더 친절한 이름이면 좋을 것 같습니다.
```suggestion
    const emailErrorElement = emailInput.nextElementSibling;
    const emailErrorContainer = emailInput.nextElementSibling;
```